app/assets/javascripts/short-form/ShortFormApplicationService.js.coffee.erb

```diff
@@ -309,6 +310,18 @@ ShortFormApplicationService = ($localStorage, $translate, $http, Upload, uuid) -
       Service.preferences["#{fileType}_error"] = true
     ))
 
+  Service.checkSurveyComplete = ->
```

Comment:
Can you clarify what this function does? Not too clear on it since I know these fields are optional